<documentation title="Space Before Casts">
    <standard>
    <![CDATA[
    Ensures there is exactly one space before a cast operator.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Single space before a cast operator.">
        <![CDATA[
$integer =<em> </em>(int) $string;
        ]]>
        </code>
        <code title="Invalid: No space before a cast operator.">
        <![CDATA[
$integer =<em></em>(int) $string;
        ]]>
        </code>
    </code_comparison>
    <code_comparison>
        <code title="Valid: Single space before a cast operator.">
        <![CDATA[
$c = $a +<em> </em>(int) $b;
        ]]>
        </code>
        <code title="Invalid: Multiple spaces before a cast operator.">
        <![CDATA[
$c = $a +<em>   </em>(int) $b;
        ]]>
        </code>
    </code_comparison>
</documentation>
